packages/inventory/src/Stocktake/api/hooks.ts

```diff
@@ -159,6 +162,64 @@ export const useSaveStocktakeLines = () => {
   });
 };
 
+export const useDeleteStocktakeLine = () => {
+  const api = useStocktakeApi();
+  return useMutation(api.deleteLines);
+};
+
+export const useDeleteSelectedLines = (): {
```

Comment:
mainly taken from outbound. 

It seems straight forward enough, though do you think this needs comments? Maybe I'm used to react-query apis and can't see that more commenting could be helpful for others?

Reply:
Nice and clear to me - and I really like the hook, and lifting the delete logic out of the Toolbar